app/main.py

```diff
@@ -1 +1,53 @@
-#  write your code here
+from __future__ import annotations
+
+
+class Animal:
+    def __init__(self, name: str,
+                 appetite: int = 0,
+                 is_hungry: bool = True) -> None:
+        self.name = name
+        self.is_hungry = is_hungry
+        self.appetite = appetite
+
+    def print_name(self) -> str:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        self.appetite = 0
+        return self.appetite
+
+
+class Cat(Animal):
+    def __init__(self, name: str,
+                 is_hungry: bool = True,
+                 appetite: int = 3) -> None:
+        appetite = 3
```

Comment:
It's a redundant line because you use a default value for appetite. You can remove this line or remove it from __init__ parameters. The same in the `Dog` class